test/customer_test.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'test_helper'
+require './test/test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.